shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -631,11 +631,11 @@ export default {
                 </div>
               </div>
               <div v-if="group.ref" class="right mr-45">
-                <template v-if="value.hasLink('update')">
+                <template v-if="value.hasLink('update') && (group.ref.canScaleDownPool() || group.ref.canScaleUpPool())">
```

Comment:
I don't think both of these canScaleX methods are there on the model - I think you need to add the missing one and have it return true by default to represent the current behvaiour.

Reply:
Correct. Thanks for pointing it out. Fixed.